.template/addons/github/.github/workflows/deploy.yml.tt

```diff
@@ -0,0 +1,75 @@
+name: Deploy
```

Comment:
should we name the file as `heroku_deploy.yml.tt` so we can support other platforms later, e.g. `aws_deploy.yml.tt`?

Reply:
hmm, as we support to deploy to Heroku by default so I used that name. It makes sense to change, but we are naming the workflows beginning with a verb, so I'll name it as `deploy_heroku.yml`